net-vpn/logmein-hamachi/logmein-hamachi-2.1.0.203.ebuild

```diff
@@ -0,0 +1,64 @@
+# Copyright 2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=6
```

Comment:
I wonder if 7 works here, namely `linux-info` should support it, but I havent tested it yet. 
Also please one empty line below EAPI to follow default ebuild skel format.

Reply:
I tried 7 and it seems to work fine, thanks